packages/driver/cypress/e2e/commands/assertions.cy.js

```diff
@@ -554,7 +551,7 @@ describe('src/cy/commands/assertions', () => {
     }, () => {
       it('should not be true', (done) => {
         cy.on('fail', (err) => {
-          expect(err.message).to.eq('expected false to be true')
+          expect(err.message).to.eq('Timed out retrying after 50ms: expected false to be true')
```

Comment:
Assertions still inherit the timeout of their parent command, but they now time out separately.

So for example, if you had

```
cy.get('button', { timeout: 1000 }).should('have.class', 'active')
```

you'd previously have 1s to both get the button and wait for it to be active. Now, you have a 1s timeout to get the button, and then a separate 1s timeout for the button to have the active class.

Reply:
I think the fact assertions still inherit timeouts is super "odd" for this command specifically.Have we talked about changing this behavior in v13 as well so assertions can own their own timeouts?